src/Microsoft.AspNetCore.Mvc.Core/ControllerBase.cs

```diff
@@ -515,6 +587,30 @@ public virtual RedirectToActionResult RedirectToAction(string actionName, string
         }
 
         /// <summary>
+        /// Redirects (<see cref="StatusCodes.Status307TemporaryRedirect"/>) to the specified action with 
+        /// <see cref="RedirectToActionResult.Permanent"/> set to false using the specified <paramref name="actionName"/>, 
+        /// <paramref name="controllerName"/>, <paramref name="routeValues"/>, and <paramref name="fragment"/>.
+        /// </summary>
+        /// <param name="actionName">The name of the action.</param>
+        /// <param name="controllerName">The name of the controller.</param>
+        /// <param name="routeValues">The parameters for a route.</param>
+        /// <param name="fragment">The fragment to add to the URL.</param>       
+        /// <returns>The created <see cref="RedirectToActionResult"/> for the response.</returns>
+        /// <remarks>At least the <paramref name="actionName"/> must be specified.</remarks>
```

Comment:
If we used to allow `null` actionName in the past for other overloads, lets not change the behavior. Do it just for this overload.